infra/recipes/terraform/azure/modules/helm_release/main.tf

```diff
@@ -0,0 +1,54 @@
+locals {
```

Comment:
Can we add in place description for the helm integration like we added in other main.tf?

Reply:
I am a bit confused by what you mean here? Do you have an example?

Reply:
Added example.